src/Template/Adrs/xml/e2b.ctp

```diff
@@ -178,17 +182,17 @@
             ?></patientsex>
             <lastmenstrualdateformat/>
             <patientlastmenstrualdate/>
-            <patientmedicalhistorytext><?php echo $adr['medical_history']; ?></patientmedicalhistorytext>
+            <patientmedicalhistorytext><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$adr['medical_history']); ?></patientmedicalhistorytext>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate